### ISSUE TITLE
FAQ: Missing/Outdated fields

### ISSUE THREAD

**Issue by Aeradriel** [wyyerd/stripe-rs#63]
I'm currently implementing Stripe on my Rust application and I found several strange things in the crate.
First of all I cannot find a `name` or `phone` field in the `Customer` or `CustomerParams` structs while it is accepted by Stripe API (https://stripe.com/docs/api/customers/create).
Moreover I noticed the `business_vat_id` field that has been changed in Stripe API for a while (cf. https://stripe.com/docs/upgrades#2018-08-23) but is still present here and not replaced with the new format.

Is the crate still maintained and usable with the current version of Stripe API and if so, maybe I missed something in the docs ?

**Comment by kestred** [comment]
@Aeradriel - Yup! The crate is maintained 😉.

For a while we were using a pretty old version of the stripe API internally, because as it turns out it changes really fast and is a lot of work to keep up to date!

**Really recently**, though I implemented automatically [generating the Stripe API from the spec](https://github.com/wyyerd/stripe-rs/pull/60) to make it a lot easier to keep it updated, without having to rely (as much) on contributors opening an issue when something as changed.

That branch _**isn't released yet**_ on `crates.io` because I'm trying to tie-up a few loose ends (e.g. https://github.com/wyyerd/stripe-rs/commit/d2d71e9a83ca2564ce805978bbbba4be62127cc1, https://github.com/wyyerd/stripe-rs/commit/f2bae33d3818f6a986ea6da476c8f6201e791ffc).

-------------------------------------------------------

As I have time to work on it, I'm trying to publish the work started in #60 as `v0.10.0` of the crate.

In the mean time, you can depend on the latest commit from `master`, which has those fields: https://github.com/wyyerd/stripe-rs/blob/master/src/resources/customer.rs#L292

e.g.
```toml
# By commit
stripe-rust = { git = "https://github.com/wyyerd/stripe-rs.git", rev = "8596bec254d0afbb3029ed76e1eb5efe2e45bfa4" }

# By branch
stripe-rust = { git = "https://github.com/wyyerd/stripe-rs.git", branch = "master" }
```

**Comment by kestred** [comment]
These changes are available as part of `v0.10.1`:

```
stripe-rust = "0.10.1"
```
